suite-native/discovery/src/discoveryConfigSlice.ts

```diff
@@ -91,27 +91,39 @@ export const selectAreTestnetsEnabled = (state: DiscoveryConfigSliceRootState) =
 export const selectDiscoveryInfo = (state: DiscoveryConfigSliceRootState) =>
     state.discoveryConfig.discoveryInfo;
 
+export const selectDiscoveryNetworkSymbols = memoizeWithArgs(
+    (state: DeviceRootState, areTestnetsEnabled: boolean) => {
+        const supportedNetworks = selectDiscoverySupportedNetworks(state, areTestnetsEnabled);
+
+        return getNetworkSymbols(supportedNetworks);
+    },
+    { size: 2 },
+);
+
 export const selectEnabledDiscoveryNetworkSymbols = memoizeWithArgs(
     (
         state: DiscoveryConfigSliceRootState & DeviceRootState & FeatureFlagsRootState,
         areTestnetsEnabled: boolean,
```

Comment:
`areTestnetsEnabled` can be forced from `prepareDiscoveryMiddleware` through `startDescriptorPreloadedDiscoveryThunk` in case we are just changing whether testnets are enabled. I'll rewrite this so it is not needed, in case you want to use what is in state already.

Reply:
bfcac19e96